### PR TITLE
added ref and set focus to new text

### PR DESCRIPTION
**Pull request by JohnAGill**
issue: [#293](https://github.com/project-vagabond/covid-green-app/issues/293), [#268](https://github.com/project-vagabond/covid-green-app/issues/268), [#306](https://github.com/project-vagabond/covid-green-app/issues/306)

### CODE REVIEW

File: src/components/atoms/markdown.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ interface Markdown {
   renderLink?: RenderLink;
   warningList?: boolean;
   renderListBullet?: (index: number, ordered: boolean, children?: any) => any;
+  markdownRef?: any;
```
Comment (floridemai) [inline src/components/atoms/markdown.tsx]: Not `any` please

File: src/components/views/upload-keys.tsx [hunk 2]
```diff
@@ -41,6 +42,11 @@ export const UploadKeys = ({navigation}) => {
   const [validationError, setValidationError] = useState<string>('');
   const [uploadToken, setUploadToken] = useState('');
   const [symptomDate, setSymptomDate] = useState('');
+  const [ref] = useFocusRef({
+    accessibilityFocus: true,
```
Comment (floridemai) [inline src/components/views/upload-keys.tsx]: `{timeout: 1000}` is enough. All the others are defaulted. Can I ask how did you choose 1000?

**Review by floridemai — COMMENTED** [review]
Some comments but this looks good.
Also, is it possible to also take care of https://github.com/project-vagabond/covid-green-app/issues/268 & https://github.com/project-vagabond/covid-green-app/issues/306 as you'll have to use the exact same approach. I'll have a look after

File: src/components/views/upload-keys.tsx (hunk 2 above)
Comment (JohnAGill) [reply src/components/views/upload-keys.tsx]: It was already being used in a few other places so I assumed it was a safe bet

**Review by floridemai — APPROVED** [review]

**Pull request merged**